content/zh/blogs/kubesphere-certificates.md

```diff
@@ -5,7 +5,7 @@ keyword: '社区, 开源, 贡献, KubeSphere'
 description: 'KubeSphere 社区管理委员会向此次获得 2020 年度评选产生的 KubeSphere Member、 KubeSphere Ambassador、KubeSphere Talented Speaker、KubeSphere Contributor 成员致谢。'
 createTime: '2021-02-11'
 author: '周鹏飞'
-snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/20210211222956.png'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-Twitter, Facebook, LinkedIncommunity/images/20210211222956.png'
```

Comment:
Are you sure this link is accessible? There are spaces in the link.

Reply:
@JiannaLiu It looks like the wrong URL is still not fixed, please use `https://pek3b.qingstor.com/kubesphere-community/images/20210211222956.png`.